dune

```diff
@@ -42,3 +42,5 @@
  (name runtest)
  (package coqide-server)
  (deps test-suite/summary.log))
+
+(dirs (:standard _build_ci))
```

Comment:
What does this do?

Reply:
https://dune.readthedocs.io/en/latest/dune-files.html#dirs-since-1-6

Reply:
I agree this should be moved elsewhere.

Reply:
This was still not removed if I'm not mistaken.